src/embed/components/Embed.tsx

```diff
@@ -81,6 +81,7 @@ const Embed: FC<UserProps> = ({ commonUser }) => {
 				poster={getFlowPlayerPoster(undefined, embedCode.content as ItemSchema)}
 				item={embedCode.content as ItemSchema}
 				canPlay={true}
+				initFlowPlayerWithoutAutoPlay={true}
```

Comment:
Maybe a more clear name would be:
```suggestion
				placeholder={false}
```
Where placeholder would default to true, so we don't affect existing flowplayers already used by AVO